.github/networks/mainnet/.env-ethereum-mainnet

```diff
@@ -0,0 +1,12 @@
+# Operational settings
+RPC_URL=https://eth.drpc.org
+
+# Uniswap
+UNIVERSAL_ROUTER_ADDRESS=0x66a9893cc07d91d95644aedd05d03f95e1dba8af
```

Comment:
I noticed that we're using the `UniversalRouterV1` address in some places and `UniversalRouterV2` address in other places. I checked and our Action is compatible with both versions but we should just keep in mind that there are slight differences. I know that the `UniswapV3ExactInputAction` I manually deployed on `eth-sepolia` and `eth-mainnet` both use the `UniversalRouterV1` address so we should change this one to the V1 address since it's already deployed. For the other chains we can deploy with the V2 address no problem but we would have to redeploy on `eth-sepolia` and `eth-mainnet` to use V2 there

ref: https://github.com/Uniswap/universal-router/blob/main/deploy-addresses/mainnet.json